nipype/interfaces/afni/preprocess.py

```diff
@@ -3013,3 +3010,56 @@ class QualityIndex(CommandLine):
     _cmd = '3dTqual'
     input_spec = QualityIndexInputSpec
     output_spec = QualityIndexOutputSpec
+
+
+class NotesInputSpec(AFNICommandInputSpec):
+    in_file = File(desc="input file to 3dNotes",
+                   argstr="%s",
+                   position=-1,
+                   mandatory=True,
+                   exists=True,
+                   copyfile=False)
+    add = traits.Str(desc="note to add",
```

Comment:
Please, do not use Str from traits:

1. Add import of Str from nipype.interfaces.base:
 ```from ..base import (..., Str)```

2. Define these inputs with the imported name:

```
add = Str(desc...)
```